src/main/resources/assets/capabilityproxy/recipes/recipes.xml

```diff
@@ -27,6 +27,33 @@
 		</output>
 	</recipe>
 
+	<recipe type="shaped">
+		<condition type="config">entity_capability_proxy</condition>
+		<condition type="config">capability_proxy</condition>
+		<input>
+			<grid>
+				<gridrow>
+					<item>minecraft:ender_eye</item>
+					<item>minecraft:end_rod</item>
```

Comment:
What about using chorus instead of an end rod? Might be a bit easier for players to obtain, but still require access to the end.

Reply:
Sure.